hydradx/notebooks/Stableswap/RepeggingToken.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
Getting errors when I run this notebook related to undefined "total_supply" variable

Reply:
removed for now, although I still want to understand the failure of the arbitrageur on these scenarios